core/src/main/scala/akka/kafka/internal/ConsumerStage.scala

```diff
@@ -153,12 +153,11 @@ private[kafka] object ConsumerStage {
   final class CommittableOffsetBatchImpl(val offsets: Map[GroupTopicPartition, Long], val stages: Map[String, Committer])
       extends CommittableOffsetBatch {
 
-    override def updated(committableOffset: CommittableOffset): CommittableOffsetBatch = {
-      val partitionOffset = committableOffset.partitionOffset
-      val key = partitionOffset.key
-
-      val newOffsets = offsets.updated(key, committableOffset.partitionOffset.offset)
-
+    private def updatedStages(
+      existing: Map[String, Committer],
```

Comment:
Think I had originally meant for this to be an aggregation and not use stages, but it makes more sense just to use stages as implemented.